installer/runner.sh

```diff
@@ -81,9 +81,17 @@ phone_home_url=$(jq -r .phone_home_url "$metadata")
 trap fail EXIT
 
 service docker start
-until docker info; do
-	sleep 3
+for i in $(seq 5); do
+	echo "Waiting for docker to respond (${i})..." &&
+		docker info &&
+		break
+	sleep 5
 done
+if ! docker info; then
+	cat /var/log/docker.log
+	echo "Docker not responding, dropping to a shell..."
```

Comment:
is this installer/runner.sh file manually executed by a human?

is it automatically executed by a machine? if so, this should not drop into a shell.

Reply:
Runs automatically, drops to a shell when an issue happens and support and/or devs are able to poke around and debug.

Reply:
Isn't this executed in a CI environment? Without user access.